WordPress/Classes/ViewRelated/Views/List/ListTableHeaderView.swift

```diff
@@ -47,12 +49,12 @@ class ListTableHeaderView: UITableViewHeaderFooterView, NibReusable {
         }()
 
         // configure title label
-        titleLabel.font = Style.sectionHeaderFont
-        titleLabel.textColor = Style.sectionHeaderTitleColor
+        titleLabel.setStyle(.bodyLarge(.emphasized))
+        titleLabel.textColor = .DS.Foreground.primary
 
         // configure separators view
         separatorsView.bottomColor = Style.separatorColor
-        separatorsView.bottomVisible = true
+        separatorsView.bottomVisible = false
```

Comment:
The `ListTableHeaderView` is used in the `CommentsViewController`, so these changes will be applied to that screen as well, and I don't think we want that.

Reply:
Sounds good, I'll revert it.